node/nanoexpress/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12.6.0
+
+RUN npm -g install pm2
+
+WORKDIR /usr/src/app
+
+COPY app.js package.json ./
+
+RUN apk update && apk add --no-cache git
+RUN npm install
+RUN ln -s /lib/libc.musl-x86_64.so.1 /lib/ld-linux-x86-64.so.2
```

Comment:
why https://www.musl-libc.org/ ?

Reply:
Because of `slim` version of Node.js image. Minimal size, faster download (really matters for me)

Reply:
rather use the official image

Reply:
Yea, I removed this line (but cannot test due of traffic, sorry)